packages/apollo-gateway/src/utilities/createSHA.ts

```diff
@@ -0,0 +1,10 @@
+import isNodeLike from './isNodeLike';
+
+export default function(kind: string): import('crypto').Hash {
```

Comment:
@abernix should we export this from `apollo-server-env` and import it from there?

Reply:
Yes!  And the same for `isNodeLike`, probably?

(Though both of these should probably move to `apollo-env` longer term, `apollo-server-env` still exists today because it still provides some Node.js 6 compatibility and we haven't merged it with `apollo-env`.)